### PR TITLE
Redis patch schedule supports "Everyday" and "Weekend" as valid DayOfWeek

### PR DESCRIPTION
**Pull request by SiddharthChatrolaMs**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@SiddharthChatrolaMs__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sichatro). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@SiddharthChatrolaMs please update assemblyinfo.cs file as well with the new version.

File: src/ResourceManagement/RedisCache/Microsoft.Azure.Management.Redis/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -20,8 +20,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Redis Cache Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure redis cache management operations.")]
 
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.1.0.0")]
```
Comment (hovsepm) [inline src/ResourceManagement/RedisCache/Microsoft.Azure.Management.Redis/Properties/AssemblyInfo.cs]: this should remain 4.0

**Review by hovsepm — CHANGES_REQUESTED** [review]

**Comment by SiddharthChatrolaMs** [comment]
Fixed all code review comments

**Review by hovsepm — APPROVED** [review]

**Pull request merged**